app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -708,6 +728,42 @@ class StatePlayerRecyclerViewAssembler private constructor(
     }
   }
 
+  private fun createBannerConfetti() {
+    val x = bannerConfettiView!!.width / 3
+    val y = bannerConfettiView!!.height / 2
+    val colorsList = confettiColors.map { getColor(bannerConfettiView.context, it) }
```

Comment:
Maybe to simplify things, we could require the caller to pass in a list of colors that it is responsible for fetching via resources directly. That might simplify your Bazel setup, and it provides a bit more flexibility in customizing the confetti experience.

Reply:
Done